core/metrics.go

```diff
@@ -31,15 +31,17 @@ const NutsMetricsPrefix = "nuts_"
 // Metrics are exposed on /metrics, by default the GoCollector and ProcessCollector are enabled.
```

Comment:
Wrong comments

Reply:
Why is it wrong?

Reply:
Must start with func/struct/interface name